packages/flutter_tools/.analysis_options

```diff
@@ -7,12 +7,16 @@
 analyzer:
   language:
     enableSuperMixins: true
-  strong-mode: false #TODO(pq): enable once strong-mode runs clean
+  strong-mode: true
   errors:
     # we allow overriding fields (if they use super, ideally...)
     strong_mode_invalid_field_override: ignore
     # we allow type narrowing
     strong_mode_invalid_method_override: ignore
+    strong_mode_static_type_error: ignore
+    strong_mode_down_cast_composite: ignore
+    type_argument_not_matching_bounds: ignore
```

Comment:
I think this may drop some potentially useful feedback on the floor (not just strong-mode provoked).  For example, looking at our test cases for this error code, I think it would suppress a warning on the following snippet:

```
class A {}
class B {}
class G<E extends A> {
  const G();
}
f() { return const G<B>(); } //<= should flag B
```
See here: https://dartpad.dartlang.org/fe9b7876ca94a9740916

cc @bwilkerson for more thoughts

If there aren't a lot of these errors it may be safer to ignore them specifically in the source?